src/styles/main.scss

```diff
@@ -1,2 +1,3 @@
 @import "./bootstrap";
+@import "./standardtable";
```

Comment:
Let's move this style to the existing `content.scss` file per [this discussion](https://github.com/openstax/k12-apps-raise/pull/50#discussion_r999531259).